pyaedt/modeler/cad/Primitives.py

```diff
@@ -4924,6 +4924,70 @@ def import_spaceclaim_document(self, input_file):
         self.refresh_all_ids()
         return True
 
+    def discovery_import(self, input_file):
+        """Import a Discovery file.
+
+        Parameters
+        ----------
+        input_file :
+            Full path and name of the Discovery file.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.CreateUserDefinedModel
+        """
+        version = settings.aedt_version[-4:].replace(".", "")
+
+        aedt_install_dir = os.environ.get("ANSYSEM_ROOT{}".format(version), "")
```

Comment:
@IreneWoyna. Why do you need to check the aedt install dir and the ansys_install_dir?
the aedt_install_dir is not even used.
Bear in mind that you can get it directly from settings as:
`settings.aedt_install_dir`
without even getting the version as you are doing.

Reply:
@gmalinve thanks for your comments! AEDT and Discovery versions need to be the same, that's why I'm checking it. But surely there's a smarter way to do it.

Reply:
In misc/misc.py there is a function to get all AEDT installed versions on the system, it retrurns a dictioary containing their installation paths (it is used in PyAEDT initialization). It is called `installed_version`. 
I suggest to create a similar function to get all Discovery installations and then just check if `settings.aedt_version` in the list of Disco installations.
Keep in mind that checking if Discovery is installed is a bit tricky because a user can install Ansys structures without Discovery, so it would have the environment variable, but Disco won't be installed.